Change labels to use "Edit"

rebeccaalpert opened the pull request:
Fixes [CONSOLE-623](https://jira.coreos.com/browse/CONSOLE-623).

Updated labels in dropdowns to use "edit" rather than "modify" or nothing.

Before:
![screen shot 2018-08-06 at 9 38 57 am](https://user-images.githubusercontent.com/7014965/43719759-94e18498-995c-11e8-8994-b4bbf61cec9c.png)

After:
![screen shot 2018-08-06 at 9 32 36 am](https://user-images.githubusercontent.com/7014965/43719456-c4aa75d2-995b-11e8-97fb-4e448a08c9be.png)

I also changed some of the modals' titles to match; let me know if that's a terrible idea and I'll switch them back.

Before:
![screen shot 2018-08-06 at 9 37 21 am](https://user-images.githubusercontent.com/7014965/43719698-65d893ee-995c-11e8-80a1-c7390db1a39e.png)

After:
![screen shot 2018-08-06 at 9 33 56 am](https://user-images.githubusercontent.com/7014965/43719506-e1c09912-995b-11e8-9b58-64d971c67030.png)

@openshift/team-ux-review - can you please review this? Thanks!

rebeccaalpert commented:
/retest

rebeccaalpert commented:
jenkins rebuild

rebeccaalpert commented:
jenkins rebuild

rebeccaalpert commented:
jenkins rebuild

rebeccaalpert commented:
Thanks! Sometimes the modal labels include the resource name (i.e. Edit Daemon Set Labels) -- do you want to keep it like that? The action menu label is always just "Edit Labels" or similar.

rebeccaalpert commented:
Brie's PR says no ellipses: https://github.com/openshift/openshift-origin-design/pull/96

I can axe them.

spadgett commented:
> Brie's PR says no ellipses

Makes sense to do it one time if we have agreement.

cc @robszumski since he asked about it in the design review

robszumski commented:
I think we're in agreement

serenamarie125 commented:
That'd be great if we can get it done all at once!

@rebeccaalpert I'd suggest that it match the menu item label.  

There may be some special cases where we may want to provide more context on the modal title label, but didn't because the string would be too long for the menu, but I'd think that would be the exception rather than the rule.

rebeccaalpert commented:
I adjusted the modal titles so they match the menu item labels and made the changes outlined in Bree's PR (removed ellipses and dropdown gear icon).

"Revision History" and "Update Strategy" were weird outliers with no "Modify" in front, so I added "Edit" to them. I can switch it back if that doesn't make sense for them; just let me know.

spadgett commented on frontend/public/components/job.jsx:
```diff
@@ -25,7 +25,7 @@ spec:
 
 const ModifyJobParallelism = (kind, obj) => ({
-  label: 'Modify Parallelism...',
+  label: 'Edit Parallelism...',
```
Remove `...`

rebeccaalpert commented:
I removed the set of ellipses I missed.

spadgett commented:
Thanks @rebeccaalpert, please squash

rebeccaalpert commented:
Squashed!

spadgett commented:
jenkins rebuild
